README.md

```diff
@@ -85,18 +116,71 @@ https://github.com/uggla/cloud_native_app
 
  * Video that will present the CICD of the application in a new empty tenant.
 
-## Teachers support
+ * Your software factory should be ready to deliver a patch or a feature
+    requested by your customer.
+    This delivery will be in production in less than a day.
+    The feature or fix requested can come from the opensource project.
+
+*Note*: A complex feature or bug fix could be delivered several times,
+improving the application between each delivery. Your customer do not
+require this to be delivered just once. They prefer to see changes frequently.
+  But the overall application service must not be impacted by such update.
+
+  In the context of this training, no complex feature or bug fix will be
+  requested by your teachers.
+
+## Teachers interaction with students
+
+
+### Hardware
+
+Teachers will provide a remote access (though VPN) to :
+
+* the *customer production platform*
+    (2 servers pre-installed with a CentOS 7 distribution)
+    Your development team will have to setup them with an OpenStack as
+    requested by the customer
+    Your development team will deploy the customer application and application
+    health control on it.
+
+* a runnable *company Openstack tenant* to install your software factory.
+    In this factory, you will be able to
+    * develop anything requested by your customers, or proposed by your team
+        for improvement/bug fix.
+    * test your application and his delivery
+    * control the application and software factory health
 
-Teachers will provide a remote access to the production platform (set of servers preinstalled with a CentOS 7 distribution) that the team will have to setup from an OpenStack perspective and deliver teams could deploy their application on it. Teachers will also provide a pre-production platform running an OpenStack distribution as a development environment for the application developers for testing purposes.
 
-A mailing list has been created to provide remote support to students during the whole period. Various people will so be available to answer questions and help with regards to platform setup and access. The mailing-list address is ensimag-openstack@lists.osp.hpe.com
+### Roles
+
+Your teachers will take and behave with 2 different roles:
+
+- *Customer representative*.
+    He will support the 2 Productions system that you will use to deliver
+    your application.
+    <br>to contact your customer: ...
+
+- *Your company IT* who host your openstack tenant.
+    <br>To contact your company IT team: ...
+
+When you contact *your customer* or *your company*, remind your group number.
+
+### Training support
+
+Outside of these roles that your teacher will have, you can get support on
+the overall training subject, by mail or during the different sessions, to:
+* Ask question,
+* Get help (VPN Access, platform setup, expertise)
+* Discuss
+
+The mailing list is:  ensimag-openstack@lists.osp.hpe.com
 
```

Comment:
Ok, but simplify what's upper, as myself I hardly got it ;-)

Reply:
to discuss